package.json

```diff
@@ -1,33 +0,0 @@
-{
```

Comment:
This should not be removed.

Reply:
Apologies, that was a mistake on my part.... on it